src/containers/Tenant/Diagnostics/TopQueries/columns/columns.tsx

```diff
@@ -27,7 +27,7 @@ const cpuTimeUsColumn: Column<KeyValueRow> = {
     render: ({row}) => formatToMs(parseUsToMs(row.CPUTimeUs ?? undefined)),
     width: 120,
     align: DataTable.RIGHT,
-    sortable: false,
+    sortable: true,
```

Comment:
`sortable` is for places, where column is used without preparation (`getTenantOverviewTopQueriesColumns`), in case of top queries sortable on not sortable state is defined by `isSortableTopQueriesColumn` in `getTopQueriesColumns` and `getRunningQueriesColumns`:
```
  return columns.map((column) => ({
        ...column,
        sortable: isSortableTopQueriesColumn(column.name),
    }));
```

Now we have sort in table, where we always display top-5 ("Top queries by cpu time, executed in the last hour")